.github/workflows/test-suite.yml

```
@@ -14,7 +14,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13", "3.14.0-alpha - 3.14"]
```

We can simplify this (`allow-prereleases: true` is set below):

```suggestion
        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13", "3.14"]
```